Made AboutPage more responsive and added some new people

SithuSoe04 opened the pull request:
Made the AboutPage more responsive by having two columns on phone screen and added some more people to the page.
<img width="349" alt="Screenshot 2024-02-05 at 8 50 12 PM" src="https://github.com/Will-Hsu/cses_webdev/assets/63730799/ac6bcad6-2b73-44f8-92c2-639864d3ff6d">

Will-Hsu reviewed (changes requested):
Great job so far! I like the two-column structure, which makes the page more visually pleasing. I noticed that the margins on the about page are not consistent with the other pages. It would be great if all pages could have standardized margins (see below). Also, the margin becomes 0 on small mobile devices.

![Screenshot 2024-02-06 at 9 47 48 AM](https://github.com/Will-Hsu/cses_webdev/assets/46422881/b353854a-b221-4dec-a372-811243101df8)
![Screenshot 2024-02-06 at 9 48 14 AM](https://github.com/Will-Hsu/cses_webdev/assets/46422881/c2a8253a-679d-4d26-8053-9fdfc2aa3aea)
![Screenshot 2024-02-06 at 9 56 17 AM](https://github.com/Will-Hsu/cses_webdev/assets/46422881/6fc957de-fc38-4b38-9504-cc38ec6da909)

SithuSoe04 commented:
I have modified the margins on the About page to align with the margins on the other pages. Additionally, I have addressed the margin becoming 0 between the photos on small devices!
<img width="352" alt="Screenshot 2024-02-10 at 4 40 49 PM" src="https://github.com/Will-Hsu/cses_webdev/assets/63730799/fcb0816d-89d2-4875-8da8-6c1092d4cfb2">
<img width="353" alt="Screenshot 2024-02-10 at 4 41 00 PM" src="https://github.com/Will-Hsu/cses_webdev/assets/63730799/b9e0f9f2-beee-4ae7-9faf-4c8a72b810b2">
<img width="308" alt="Screenshot 2024-02-10 at 4 41 14 PM" src="https://github.com/Will-Hsu/cses_webdev/assets/63730799/d57316f1-9608-4f70-8b54-decec422d693">
